S04 us10 volume bar

MeertAdrien opened the pull request:
added a volume bar

Salvak613 commented on client/src/components/VolumeBar.tsx:
```diff
@@ -0,0 +1,47 @@
+
+  useEffect(() => {
+    if (player) {
+      console.log(`Changement de volume : ${volume * 100}%`);
```
remove console log

Benja-san reviewed (commented):
need to discuss some points of this us : 
- handleVoilumeChange lines
- VolumeBar component is not imported anywhere ?

MeertAdrien commented:
I see what you mean.
The VolumeBar component isn't actually called in the SpotifyPlayer file.
I'll fix that.

MeertAdrien commented:
Updated the volume bar that does not reset the % of the sound for each new title. Removed const handleVolumeChange from SpotifyPlayer because it was never called.
